packages/cli/src/databases/repositories/settings.repository.ts

```diff
@@ -9,6 +9,24 @@ export class SettingsRepository extends Repository<Settings> {
 		super(Settings, dataSource.manager);
 	}
 
+	async dismissNewBanner({ bannerName }: { bannerName: string }): Promise<{ success: boolean }> {
```

Comment:
since we won't end up with dozens of banners, maybe we can use stronger typing for `bannerName`, so that it only allows a fix set of values ?

Reply:
We can use [Banners enum](https://github.com/n8n-io/n8n/blob/8dee720f014def42e04cf3fd68842ba4a60d3b63/packages/editor-ui/src/constants.ts#L572) here, yeah.